src/ui/profile_preferences.ui

```diff
@@ -1216,7 +1216,7 @@ you can use it but there could be issues with aligning columns of text</string>
       <attribute name="title">
        <string>Editor</string>
       </attribute>
-      <layout class="QVBoxLayout" name="verticalLayout_tab_codeEditor">
+      <layout class="QVBoxLayout" name="verticalLayout_2">
```

Comment:
Unwanted change IMHO - "2" is not a useful identifier given how many `QVBoxLayout`s there are in this file, whereas "tab_codeEditor" though long, IS, (and it is only used within this file and not referred to by our code beyond that at all)...
```suggestion
      <layout class="QVBoxLayout" name="verticalLayout_tab_codeEditor">
```